build/config-reference-v2.json

```diff
@@ -4,6 +4,10 @@
     "type": "ENCRYPTED_JSON",
     "encryptedJson": "<WALLET>"
   },
+  "documentStorage": {
+    "apiKey": "randomKey",
+    "url": "https://tradetrust-functions.netlify.app/.netlify/functions/storage"
```

Comment:
this the new url? we not using aws lambda anymore?

Reply:
yup this will be the new url